meta-oe/recipes-bsp/lm_sensors/lmsensors_3.6.0.bb

```diff
@@ -10,12 +10,15 @@ DEPENDS = " \
     virtual/libiconv \
 "
 
+PV = "3-6-0+git${SRCPV}"
```

Comment:
this should be  `3.6.0+git${SRCPV}`

Reply:
In fact, `3-6-0` is the release tag from lm-sensors [1], with 3.6.0, I raised the following issue:

```
INFO: Skip package lmsensors (status = UNKNOWN_BROKEN, current version = 3.6.0, next version = 3-6-0)
```

[1]  https://github.com/lm-sensors/lm-sensors/releases

Reply:
what happens when you set `PE = "1" ` instead of setting PV ?

Reply:
Same, but anyway, I will drop this patch from this series ;).
I will investigate more, and I'll send another PR.